WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/interval/intervaltypes/IDEActiveInterval.java

```diff
@@ -0,0 +1,15 @@
+package nl.tudelft.watchdog.core.logic.interval.intervaltypes;
```

Comment:
It evolved from EclipseActiveInterval and IntelliJActiveInterval. I think it makes sense to have it defined what is Active/Open.